src/app/SecurityPanel/StoreJmxCredentials.tsx

```diff
@@ -141,18 +141,26 @@ export const StoreJmxCredentials = () => {
     addSubscription(forkJoin(tasks).subscribe());
   };
 
-  const handleModalClose = () => {
+  const handleAuthModalClose = () => {
```

Comment:
`useCallback()`?

Reply:
I didn't even write this one, I just noticed that it had a very similar name to what I was doing :laughing: , but yeah I will change it.

Reply:
Sorry again :-) I like to follow the boyscout's rule: "Always leave the code better than you found it."

Since you're working in this area at the moment, I'm holding you responsible for cleaning up things I see at the edges of the ~~campsite~~ diffs ;-)